src/components/structures/login/Registration.js

```diff
@@ -426,7 +426,12 @@ module.exports = React.createClass({
         return (
             <div className="mx_Login">
                 <div className="mx_Login_box">
-                    <LoginHeader icon={this.state.teamSelected ? this.state.teamSelected.icon : null}/>
+                    <LoginHeader
+                        icon={this.state.teamSelected ?
+                            "https://riot.im/rts/static/common/" +
+                            this.state.teamSelected.domain + "/icon.png" :
+                            null}
+                    />
```

Comment:
Hang on, surely these should be hosted on the RTS? Otherwise each install of Riot now has to have an icon for each team it needs to support.